go-controller/pkg/ovn/libovsdbops/util.go

```diff
@@ -12,7 +13,7 @@ func TransactAndCheck(client client.Client, ops []ovsdb.Operation) ([]ovsdb.Oper
 		return []ovsdb.OperationResult{{}}, nil
 	}
 
-	results, err := client.Transact(ops...)
+	results, err := client.Transact(context.TODO(), ops...)
```

Comment:
@dcbw Should we use context.Background() instead?

Reply:
@flavio-fernandes I think TODO is correct here, because we'll want to (eventually) pass a context down to this function from somewhere, we just don't have that yet.